yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -60,6 +62,9 @@ private[yarn] class YarnAllocator(
 
   import YarnAllocator._
 
+  // RackResolver logs an INFO message whenever it resolves a rack, which is way too often.
+  Logger.getLogger(classOf[RackResolver]).setLevel(Level.WARN)
```

Comment:
So won't this override any user supplied loglevel for RackResolver? What if the user legitimately wants to see the messages?